assets/lang/pt_BR.ini

```diff
@@ -161,7 +161,7 @@ Direct3D9 = Direct3D &9
 Direct3D11 = Direct3D &11
 Disassembly = &Disassembly...
 Discord = Discord
-Display Layout && Effects = Exibir o editor do esquema...
+Display Layout Editor = Exibir o editor do esquema...
```

Comment:
It looks like this might be a mistake?  There's a couple others here, like Non-Buffered Rendering?

-[Unknown]

Reply:
@unknownbrackets - you are right, i paid attention to the entries removed/added in eng file and missed those ones, i fixed those and changed others for the better, thanks for telling me.

Sorry Hrydgard for that.